buda/entities/payment_terms_and_schedule.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package buda.entities;
```

Comment:
Sorry, this is something I should have given you a heads up on @mallen5311, but we might need to change the package names. 

In order for the Node.js and Python packages to work with the protobuf files, we had to have all the Messages and Services share a single namespace, so we're just using 

```protobuf
package buda;
```